coss/opensource_clubs/templates/blocks/activity_card.html

```diff
@@ -0,0 +1,16 @@
+{% load static wagtailcore_tags wagtailimages_tags %}
+
+<div class="card card-mini">
+  {% if value.image %}
+    {% image value.image fill-180x180 class="rounded-circle card-img" %}
+  {% else %}
+    <img src="{% static 'img/clubs-illustration.svg'%}" class="rounded-circle card-img" alt="activity">
+  {% endif %}
+  <h4 class="card-header">{{ value.title }}</h4>
+  <div class="card-body">
+    {{ value.text|richtext }}
```

Comment:
Is there a specific reason on why don't you use the text input block here and you are re-adding these entries?